relationships/synthesis/APM-APPLICATION-to-INFRA-AWSMSKTOPIC.yml

```diff
@@ -0,0 +1,56 @@
+relationships:
+  - name: apmProducesAwsMskTopic
+    version: "1"
+    origins:
+      - APM Metrics
+    conditions:
+      - attribute: metricName
+        regex: "^MessageBroker/Kafka/Nodes/[^/]*\\.amazonaws\\.com:9092/Produce/.*"
```

Comment:
I think the port depends on the used protocol.
Would it make sense to change that to 9XXX ?

https://docs.aws.amazon.com/msk/latest/developerguide/port-info.html